modules/tinymce/src/plugins/lists/test/ts/browser/ContentEditableFalseActionsTest.ts

```diff
@@ -0,0 +1,102 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/lists/Plugin';
+
+describe('browser.tinymce.plugins.lists.ContentEditableFalseActionsTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'lists',
+    toolbar: 'numlist bullist',
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Plugin ], true);
+
+  it('TINY-8920: List toolbar buttons are disabled when in noneditable OL list', () => {
+    const editor = hook.editor();
+    const content = '<ol contenteditable="false">\n' +
+      '<li contenteditable="true">editable</li>\n' +
+      '<li>noneditable</li>\n' +
+      '<li contenteditable="true">editable</li>\n' +
+    '</ol>';
+    editor.setContent(content);
+    TinySelections.setCursor(editor, [ 1, 2, 0 ], 0);
+    TinyUiActions.clickOnToolbar(editor, 'button[aria-label="Numbered list"]');
```

Comment:
Why are you trying to click on something that's supposed to be disabled? We shouldn't be doing that as it's not actually checking it's disabled, all you're doing here is checking that the button does nothing which isn't the same thing.

Reply:
How would I check that the button is disabled?